### PR TITLE
Add suuport for AWS  Bedrock Cohere embedding

### PR DESCRIPTION
**Pull request by kerlion**
# Description

I missed 2 changed files in last PR, change 2 files, and added 2 files. Added support below 2 Cohere embedings.
- cohere.embed-english-v3
- cohere.embed-multilingual-v3

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Tested in my docer ENV Dify 0.6.2.

- [ ] TODO

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by crazywoola** [comment]
Please fix the lint error. :)

**Comment by kerlion** [comment]
> Please fix the lint error. :)

I have fixed the import order.

File: api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-english-v3.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+model: cohere.embed-english-v3
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-english-v3.yaml:1]:
```suggestion
model: cohere.embed-english-v3.0
```

File: api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml [hunk 2]
```diff
@@ -1 +1,3 @@
 - amazon.titan-embed-text-v1
+- cohere.embed-english-v3
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml:2]:
```suggestion
- cohere.embed-english-v3.0
```

File: api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml [hunk 3]
```diff
@@ -1 +1,3 @@
...
+- cohere.embed-multilingual-v3
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml:3]:
```suggestion
- cohere.embed-multilingual-v3.0
```

File: api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-multilingual-v3.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+model: cohere.embed-multilingual-v3
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-multilingual-v3.yaml:1]:
```suggestion
model: cohere.embed-multilingual-v3.0
```

File: api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-multilingual-v3.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+model_type: text-embedding
+model_properties:
+  context_size: 512
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-multilingual-v3.yaml:4]: https://github.com/langgenius/dify/blob/a18dde9b0dda6a32d87903c00da41e5211a5901c/api/core/model_runtime/model_providers/cohere/text_embedding/embed-multilingual-v3.0.yaml#L4

See this. `max_chunks`

File: api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-english-v3.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+model_type: text-embedding
+model_properties:
+  context_size: 512
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-english-v3.yaml:4]: https://github.com/langgenius/dify/blob/a18dde9b0dda6a32d87903c00da41e5211a5901c/api/core/model_runtime/model_providers/cohere/text_embedding/embed-english-v3.0.yaml#L4
See this and there seems to be a `max_chunks` as well.

**Review by crazywoola — CHANGES_REQUESTED** [review]
Don't forget to rename these yaml files' name to `v3.0.yaml`

**Comment by kerlion** [comment]
> Please fix the lint error. :)

Hi, the **Bedrock Cohere name** is a little different from **Cohere name**. I have tested it before , everything works fine.

**Comment by kerlion** [comment]
> Please fix the lint error. :)

We cannot do the change like that, I tested it with name like  xxxv3.0 today, it will reprot below error.

Arguments: (InvokeError(),)
[2024-04-15 07:46:11,295: ERROR/MainProcess] consume document failed
Traceback (most recent call last):
  File "/app/api/core/model_runtime/model_providers/bedrock/text_embedding/text_embedding.py", line 226, in _invoke_bedrock_embedding
    response = bedrock_runtime.invoke_model(
  File "/usr/local/lib/python3.10/site-packages/botocore/client.py", line 535, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/local/lib/python3.10/site-packages/botocore/client.py", line 983, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the InvokeModel operation: Could not resolve the foundation model from the provided model identifier.

**Comment by kerlion** [comment]
@guchenhe, @crazywoola  PLS review it.

**Review by guchenhe — APPROVED** [review]

**Pull request merged**